packages/mongodb-chatbot-server/src/routes/conversations/commentMessage.ts

```diff
@@ -0,0 +1,134 @@
+import { ObjectId } from "mongodb-rag-core";
+import {
+  Conversation,
+  ConversationsService,
+} from "../../services/ConversationsService";
+import {
+  Request as ExpressRequest,
+  Response as ExpressResponse,
+  NextFunction,
+} from "express";
+import { getRequestId, logRequest, sendErrorResponse } from "../../utils";
+import { isValidIp } from "./utils";
+import { z } from "zod";
+import { SomeExpressRequest } from "../../middleware/validateRequestSchema";
+
+export type CommentMessageRequest = z.infer<typeof CommentMessageRequest>;
+
+export const CommentMessageRequest = SomeExpressRequest.merge(
+  z.object({
+    headers: z.object({
+      "req-id": z.string(),
+    }),
+    params: z.object({
+      conversationId: z.string(),
+      messageId: z.string(),
+    }),
+    body: z.object({
+      comment: z.string().min(1, { message: "Comment cannot be empty" }),
```

Comment:
i think there should also be some max length for the comment.

not sure if it should be here at the schema level or the route level

Reply:
Added to the route config